From a4bb4ff84f4d24f5dea0b700573d07158440874e Mon Sep 17 00:00:00 2001
From: Stephen Gallagher <sgallagh@redhat.com>
Date: Wed, 9 Apr 2014 15:29:09 -0400
Subject: [PATCH] Support building with parallel-installed Django

On systems such as Fedora where older versions of Django are
installed as alternative versions (and not directly importable),
we need to ensure that we are importing exactly the right version
of the Django modules.
---
 contrib/internal/build-media.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/contrib/internal/build-media.py b/contrib/internal/build-media.py
index e317bca46bbd5c5f23075f13fb3096d254932674..fbabdb4098cb623098a4c421b78a1224548dbc82 100755
--- a/contrib/internal/build-media.py
+++ b/contrib/internal/build-media.py
@@ -3,6 +3,10 @@
 import os
 import sys
 
+import __main__
+__main__.__requires__ = ['Django>1.4,<1.5']
+import pkg_resources
+
 from django.core.management import call_command, setup_environ
 
 
-- 
1.9.0

